config/routes.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
```

Comment:
minor comment: maybe move this to the bottom of the file? Rails reads routes in order, I prefer to keep these endpoint at the bottom

Reply:
No problem, I prefer to set mount at first of the file but I can move it at end